### ISSUE TITLE
Time to update accessory status

### ISSUE THREAD

**Issue by metalic112** [jchnlemon/homebridge-telldus#39]
Not so much a bug, but how long does it take for your devices to show the status when 
1. Pulling up the control panel from bottom of the screen?
2. Launching the Apple Home app and all devices show current status?

I don't know if it's normal but mine with 15 devices can take up to 20 seconds before update is complete (both local and remote).

**Comment by mifi** [comment]
That's strange. for me it only takes a 1-2 seconds, unless i've just arrived home and my phone hasn't gotten wifi yet, then it takes probably up to20 seconds. I only have 6 devices though. maybe telldus is throttling requests, you could start homebridge in debug mode and check if something is taking too long. (see other threads for how this is done)
What do you mean by local and remote?

**Comment by metalic112** [comment]
As a matter of fact, one accessory "Fönster", which is a dimmer consistently takes 4s while everything else takes less then 100ms. Which is weird... I do run a Telldus Duo with Tellive connector to the Telldus Live service which then goes back to Homebridge. I took the Fönster lamp of my favourites and now everything updates quickly. 

I compared it to my "Taklampa" which also runs with a dimmer, its configured exactly the same in the config but that one updates really fast (less then 20 ms) so I don't know why this happens. Oh well at least now I know. Thanks for the help! 

Local and remote, I was referring to being on the home wifi or trying remotely through Apple TV/iPad
